test/uk/gov/hmrc/statepension/services/StatePensionServiceSpec.scala

```diff
@@ -1186,36 +1191,40 @@ class StatePensionServiceSpec extends StatePensionUnitSpec with OneAppPerSuite w
         NpsNIRecord(qualifyingYears = 35, List(NpsNITaxYear(2000, false, false, true), NpsNITaxYear(2001, false, false, true)))
       ))
 
-      lazy val exclusionF: Future[StatePensionExclusion] = service.getStatement(generateNino()).left.get
+      lazy val statePensionF: Future[StatePension] = service.getStatement(generateNino()).right.get
 
-      "return abroad" in {
-        whenReady(exclusionF) { exclusion =>
-          exclusion.exclusionReasons shouldBe List(Exclusion.Abroad)
+      "NOT return abroad" in {
+        whenReady(statePensionF) { statePension =>
+          statePension should not be (Nil)
```

Comment:
I'm not sure if this is testing anything

Reply:
Do you want me test a SP object something like
statePension should be 
(StatePension(Matchers.any(),Matchers.any(),Matchers.any(),Matchers.any(),Matchers.any(),Matchers.any(),Matchers.any(),Matchers.any(),Matchers.any(),Matchers.any()))

or Check values??

Reply:
Check the values